src/middleware/ErrorHandler.ts

```diff
@@ -2,13 +2,17 @@ import { NextFunction, Request, Response } from 'express';
 import { INTERNAL_SERVER_ERROR } from 'http-status-codes';
 
 // @ts-ignore
-export function handler(err: any, req: Request, res: Response, nextFunction: NextFunction): void {
+export function notFoundHandler(req: Request, res: Response, next: NextFunction): void {
+    res.status(404).render('error');
+}
+
+
+// @ts-ignore
+export function defaultHandler(err: any, req: Request, res: Response, next: NextFunction): void {
     console.error(err.message);
     if (!err.statusCode) {
         err.statusCode = INTERNAL_SERVER_ERROR;
     }
-    res.status(err.statusCode).send({
-        error: err.message
-    });
-
+    res.render('error')
```

Comment:
Does it return 500 status code? I wonder how setting `err.statusCode` influences response status code.